build.sbt

```diff
@@ -107,15 +109,16 @@ headerSources in Compile ++= {
 
 lazy val root = (project in file("."))
   .enablePlugins(NoPublish)
-// Don't forget to add your sbt module here!
-// A missing module here can lead to failing Travis test results
+  // Don't forget to add your sbt module here!
+  // A missing module here can lead to failing Travis test results
   .aggregate(
     `protocols`,
     `proxy`,
     `java-support`,
     `java-support-docs`,
     `java-support-tck`,
     `java-shopping-cart`,
+    `java-valueentity-shopping-cart`,
```

Comment:
Given that we'll seeing the value entities as the default, this implementation of the shopping cart could just be `java-shopping-cart`, with the event sourced version updated to `java-eventsourced-shopping-cart`.

We can make the switch to value entities as the default later though too.

Reply:
I will suggest to switch the value entity as default later. I will keep it mind and write an issue for it. the metrics is an open issue here and should be also done. I will also write a issue for that.

Reply:
:+1: sounds good. Issues for switching default entity and adding metrics would be useful.